ChangeLog.markdown

```diff
@@ -14,6 +14,9 @@ Current trunk code
     - Emscripten: https://github.com/kripken/emscripten/compare/1.37.13...incoming
     - Emscripten-LLVM: https://github.com/kripken/emscripten-fastcomp/compare/1.37.13...incoming
     - Emscripten-Clang: https://github.com/kripken/emscripten-fastcomp-clang/compare/1.37.13...incoming
+ - Added new html setting Module['scriptDirectory'] that allows customizing the URL where .wasm, .mem and some other files are located (defaults to the same location as .js file)
+ - If Module['locateFile'] is not specified explicitly it will be defined automatically using Module['scriptDirectory']
+ - Module['locateFile'] priority comparing to other options decreased since it is now always defined
```

Comment:
The format that is used in the ChangeLog is to first have the changes, and then at the end of each changes to have the block

```
  - To see a list of commits in the active development branch 'incoming', which have not yet been packaged in a release, see
     - Emscripten: https://github.com/kripken/emscripten/compare/1.37.13...incoming
     - Emscripten-LLVM: https://github.com/kripken/emscripten-fastcomp/compare/1.37.13...incoming
     - Emscripten-Clang: https://github.com/kripken/emscripten-fastcomp-clang/compare/1.37.13...incoming
```

so these would go four lines up.